### PR TITLE
docs: add steps in troubleshooting

### PR DESCRIPTION
**Pull request by georgekallilas**
- Move the troubleshooting steps from https://help.sentry.io/product-features/integrations/github-why-can-t-i-add-a-repository/
- Make a minor adjustment to the access level required to setup the integration

### CODE REVIEW

File: src/docs/product/integrations/source-code-mgmt/github/index.mdx [hunk 1]
```diff
@@ -275,6 +275,12 @@ For more details, see the full documentation for [Single Sign-on](/product/accou
 
 ## Troubleshooting
 
+If you’re having issues adding a GitHub repository:
+
+- Make sure you are the Owner of your GitHub organization.
+- Make sure you are either an Admin, Manager, or Owner of your Sentry organization.
+- [Disconnect](https://sentry.io/account/settings/identities/) your GitHub Identity, then [reconnect](https://sentry.io/account/settings/social/associate/github/) your GitHub Identity.
```
Comment (imatwawana) [inline src/docs/product/integrations/source-code-mgmt/github/index.mdx:282]:
```suggestion
- [Disconnect, and then reconnect](https://sentry.io/account/settings/identities/) your GitHub Identity.
```

**Review by imatwawana — APPROVED** [review]
LGTM!

File: src/docs/product/integrations/source-code-mgmt/github/index.mdx (hunk 1 above)
Comment (georgekallilas) [reply src/docs/product/integrations/source-code-mgmt/github/index.mdx:282]: The links are not the same :)

Comment (imatwawana) [reply src/docs/product/integrations/source-code-mgmt/github/index.mdx:282]: Sorry about that! How about this?

```suggestion
- [Disconnect](https://sentry.io/account/settings/identities/), and then [reconnect](https://sentry.io/account/settings/social/associate/github/) your GitHub Identity.
```

**Pull request merged**